### ISSUE TITLE
transparency issue in overlay

### ISSUE THREAD

**Issue by sriharivishnu** [TakuSemba/CropMe#17]
In CropOverlay.kt,

```
backgroundAlpha = a.getFraction(
            R.styleable.CropLayout_cropme_background_alpha,
                DEFAULT_BASE,
                DEFAULT_PBASE,
                DEFAULT_BACKGROUND_ALPHA
        ) * COLOR_DENSITY
```

should be
```
backgroundAlpha = a.getFraction(
            R.styleable.CropLayout_cropme_background_alpha,
                DEFAULT_BASE,
                DEFAULT_PBASE,
                DEFAULT_BACKGROUND_ALPHA
        )
```

**Comment by TakuSemba** [comment]
Thank you for reporting the issue.
You are absolutely right, I will fix it on v2.0.7.

**Comment by TakuSemba** [comment]
I'm closing this because v2.0.7 is released.
